lib/redis/store/interface.rb

```diff
@@ -5,8 +5,16 @@ def get(key, options = nil)
         super(key)
       end
 
+      REDIS_SET_OPTIONS = %i(ex px nx xx keepttl).freeze
```

Comment:
hmm, I kinda thought this would be gone in the diff when I merged your other branch...could you try rebasing this one against master so the diff is a bit cleaner?

Reply:
I rebased the branch, it's gone now.